### PR TITLE
 Add Dockerfile and docker-compose

### PR DESCRIPTION
**Pull request by luispuentesvega**
# PR Details

- Implement Dockerfile
- Implement docker-compose

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM node:14-alpine
```
Comment (fabiancnieto) [inline Dockerfile:1]: let's use 16

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      - dynamodb
+  dynamodb:
+    image:  amazon/dynamodb-local
+    container_name: my-dynamodb
+    hostname: dynamodb
+    restart: always
+    volumes:
+      -  ./my-dynamodb-data:/home/dynamodblocal/data
+    ports:
+      - 8000:8000
```
Comment (fabiancnieto) [inline docker-compose.yml:17]:
```suggestion
      - "8000:8000"
```

File: Dockerfile (hunk 1 above)
Comment (luispuentesvega) [reply Dockerfile]: @fabiancnieto Done! Thanks!

File: docker-compose.yml (hunk 2 above)
Comment (luispuentesvega) [reply docker-compose.yml:17]: Done!

**Review by fabiancnieto — APPROVED** [review]

**Pull request merged**